installation/linux.md

```diff
@@ -6,42 +6,51 @@ description: >-
 
 ## Download
 
-Go to [admin.edgedelta.com](https://admin.edgedelta.com) \(or contact the Edge Delta team at [info@edgedelta.com](mailto:info@edgedelta.com)\) to create an account and get access to the agent deployment portal. 
+Go to [admin.edgedelta.com](https://admin.edgedelta.com) \(or contact the Edge Delta team at [info@edgedelta.com](mailto:info@edgedelta.com)\) to create an account and get access to the agent deployment portal.
 
 ## Installation
 
-_**Note**: The following steps below can be automated by selecting the 'Deploy' button on the right-hand side of a given configuration in the Edge Delta Admin Portal. After hitting the 'Deploy' button, a dialog box will appear with a pre-configured Linux command containing the appropriate API Key for deployment. Simply run that command on the host you want to deploy Edge Delta on, and the installation / deployment process will begin._  
+_**Note**: The following steps below can be automated by selecting the 'Deploy' button on the right-hand side of a given configuration in the Edge Delta Admin Portal. After hitting the 'Deploy' button, a dialog box will appear with a pre-configured Linux command containing the appropriate API Key for deployment. Simply run that command on the host you want to deploy Edge Delta on, and the installation / deployment process will begin._
 
-Replace the &lt;YOUR\_API\_KEY&gt; field from the command below with your configuration API Key from the administration portal: 
+Replace the &lt;YOUR\_API\_KEY&gt; field from the command below with your configuration API Key from the administration portal:
 
 ![](../.gitbook/assets/screen-shot-2020-03-31-at-1.16.15-pm.png)
 
 Replace the &lt;DOWNLOAD\_URL&gt; field from the command below with the installation endpoint URL you received from the Edge Delta team.
 
 ```text
-ED_API_KEY=<YOUR_API_KEY> bash -c "$(curl -L <DOWNLOAD_URL>/install.sh)"
+sudo ED_API_KEY=<YOUR_API_KEY> bash -c "$(curl -L <DOWNLOAD_URL>/install.sh)"
 ```
 
-The installation process may prompt for the sudo password if you are not running as root. 
+The installation process may prompt for the sudo password if you are not running as root.
 
 The installation process deploys Edge Delta into the path`/opt/edgedelta/agent/` and system service `edgedelta` starts automatically with default configuration.
 
+ED_ENV_VARS special variable is used as part of the installation command to pass one or more persistent environment variables to the agent which will run as the system service.
+
+```bash
+sudo ED_API_KEY=<your api key> \
+ED_ENV_VARS="MY_VAR1=MY_VALUE_1,MY_VAR2=MY_VALUE_2" \
+bash -c "$(curl -L https://release.edgedelta.com/release/install.sh)"
+```
+
 ## Troubleshooting
 
 Check the agent's service status using one of the following commands \(dependent on Linux distribution\):
 
 ```text
+sudo su
 service edgedelta status
 systemctl status edgedelta
 /etc/init.d/edgedelta status
```

Comment:
this returns " No such file or directory" on my linux

Reply:
Yeah it lists 3 alternative commands depending on linux distro as described in line 43.

Reply:
I removed, sudo su it is confusing there,  added sudo to each command.